docs/HowTo/Withdrawal-Keys.md

```diff
@@ -0,0 +1,85 @@
+---
+title: How to update your withdrawal credentials
+---
+
+# Update your withdrawal credentials
+
+When you create a validator, it’s currently possible to set its [withdrawal](../Concepts/Withdrawals.md) address
+to a BLS address, or an Ethereum address. After the Capella fork, users must use an Ethereum address as their withdrawal
+address to allow automatics transfers.
```

Comment:
typo `automatics`

Reply:
Once capella is active, I couldn't find anything that would actually restrict a user from creating a 0x00 withdrawal key, they just wont accrue rewards if they do... Maybe we just delete the whole last sentence?

Reply:
@rolfyone, can I update this to, "After the Capella upgrade, only Ethereum adresses will accrue rewards automatically."?

Reply:
... no that sounds bad...

Reply:
fixed.